themes/default/layouts/page/kubecon.html

```diff
@@ -1,106 +1,137 @@
 {{ define "hero" }}
-    {{ partial "hero.html" (dict "title" .Params.title) }}
+    <header class="page-hero">
+        <div class="dot-background-container" style="background-color: #cff1fb">
+            <img class="mx-auto w-full" src="/images/kubecon/kubecon-hero-2023.png" />
+        </div>
+        <div class="page-hero-title">
+            <h1 class="small-title w-1/3">
+                <span><span style="font-size: 2.5rem" data-text="{{ .Params.title }}">{{ .Params.title }}</span></span>
+            </h1>
+        </div>
+    </header>
 {{ end }}
 
 {{ define "main" }}
-    <section id="ctas" class="container mx-auto my-16 text-center">
-        <h2>Interact with us at KubeCon + CloudNativeCon!</h2>
+    <div class="w-2/3 mt-8 text-center mx-auto">
+        <h5>We are going to KubeCon + CloudNativeCon</h5>
```

Comment:
is this text supposed to be in the main content area? or is it supposed to overlap the header background

![image](https://user-images.githubusercontent.com/16751381/218595580-8fc10387-0155-4a0a-a73d-3c1abb107d63.png)

Reply:
I think this is one of those cached styles situations, the latest update should have fixed this (works locally and in preview on my browser)